packages/messageport/example/lib/main.dart

```diff
@@ -0,0 +1,239 @@
+import 'package:flutter/material.dart';
+import 'dart:async';
```

Comment:
You can also apply the linter rules to the example files by commenting out the line
```yaml
- "**/example/**"
```
in the `analysis_options.yaml` file.

I excluded them only because we usually copy examples from the "original" package and we don't care how well they are written. Since this package is written by you from scratch, you can temporarily include the example files for analysis.